bot/exts/utilities/realpython.py

```diff
@@ -32,13 +32,16 @@ def __init__(self, bot: Bot):
 
     @commands.command(aliases=["rp"])
     @commands.cooldown(1, 10, commands.cooldowns.BucketType.user)
-    async def realpython(self, ctx: commands.Context, amount: Optional[int] = 5, *, user_search: str) -> None:
+    async def realpython(self, ctx: commands.Context, amount: Optional[int] = 5, *, user_search: str=None) -> None:
         """
         Send some articles from RealPython that match the search terms.
-
```

Comment:
I think keeping the blank line in the docstring is more consistent with the style of the code base.

Reply:
Yes this is how subject vs. descriptions are used for docstrings. Without this, it'll be interpreted as a space by many tools (like it does in Markdown and HTML).